crates/cext/cbindgen.toml

```diff
@@ -1,5 +1,8 @@
+language = "C"
 include_version = true
+include_guard = "QISKIT_H_INCLUDED"
```

Comment:
Very minor: The conventional name for include-guard variables is just the name of the file, e.g. `QISKIT_H`, not `QISKIT_H_INCLUDED`.

Reply:
Changed to `QISKIT_H` 👍🏻